### DISCUSSION TITLE
Expired sample token on the Authenticated Requests section (adding-spring-security)

### DISCUSSION THREAD

**Discussion by barbara-teruggi** [timtebeek/spring-security-workshop#101] [Q&A] [answered]
Good afternoon @timtebeek ,

I'm doing your workshop (which I've found out about in JBcnConf), and I'm stuck at the following section: https://github.com/timtebeek/spring-security-workshop/tree/main/adding-spring-security#authenticated-requests
I can't install Keycloak nor Docker Compose on my computer due to some technical limitations, so I'd need to use the sample token you provide in the documentation.
The issue is that the token is expired at the moment (was valid until August 7th). 

Would it be possible to have a new token and/or a way to generate new ones, or a token with a longer duration? 
I'd appreciate any help you could provide with on this matter. 

Thank you very much in advance and have a good weekend! #100 
BR. 
Bárbara.

**Comment by timtebeek** [comment, marked as answer]
Hi Bárbara, great to hear you're still working through the workshop. Looked into refreshing the token just now, but I need to dive in deeper after work. For now there's [a pull request you can track](https://github.com/timtebeek/spring-security-workshop/pull/102), with [a corresponding branch](https://github.com/timtebeek/spring-security-workshop/tree/renew_wiremock_stubs_and_token).

If you run the stubs in that branch it's already working for [the new token with `HR` role](https://github.com/timtebeek/spring-security-workshop/tree/renew_wiremock_stubs_and_token/adding-spring-security#authenticated-requests), but not yet for token for alice; That last part I hope to fix quickly, but didn't want to keep you waiting as you can probably work through the rest of the workshop with that HR token as well.

Hope this helps!

**Reply by timtebeek** [reply]
Okay the new tokens both work, provided you [add this ClaimSetConverter](https://github.com/timtebeek/spring-security-samples/commit/e6a5c263e976b4b9f7e3852711e16d04f931804b) to your actual implementation; now all that's left is to update the documentation in the workshop to cover this.

**Reply by timtebeek** [reply]
[Documentation also updated](https://github.com/timtebeek/spring-security-workshop/pull/102); hope it all works as expected for you as well. Let me know if you have any questions!

**Reply by barbara-teruggi** [reply]
Hi @timtebeek ! 

Wow! that was quick! I've downloaded the spring-security-samples repo and was re-adapting my code (which I also updated with your latest changes), but was overloaded with work hehe.. will resume this when I get more spare time (normally I take time on Fridays). 
I think I've got everything I need and will be able to continue :) 

Thank you VERY much for your quick response. 
Cheers!
